core/services/ocr2/plugins/ocr2keeper/config/config.go

```diff
@@ -0,0 +1,34 @@
+package config
+
+import (
+	"github.com/pkg/errors"
+)
+
+// PluginConfig contains custom arguments for the OCR2Keeper plugin.
+type PluginConfig struct {
+	LinkEthFeedAddress string   `json:"linkEthFeedAddress"`
```

Comment:
what do we need this for? if it's to include the LINK/ETH price in the report, then I think we want to fetch that offchain (better liveness).
![image](https://user-images.githubusercontent.com/14809513/182693147-2f4cd3a6-1efc-42fe-9779-f931a67c2169.png)

Reply:
also, can we call it something like LinkNative_ since we operate on more chains than eth, and that sometimes gets confusing

Reply:
What does observing an EA entail? If it adds complexity in deployment and adds a dependency on external API then I think it would be easier to just depend on onchain feed atleast in mvp